### PR TITLE
Fix xliff-tasks entry in Version.Details.xml

### PR DESCRIPTION
**Pull request by akoeplinger**
xliff-tasks was merged into arcade in 9.0
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/2208)

### CODE REVIEW

**Review by ViktorHofer — APPROVED** [review]

**Comment by ViktorHofer** [comment]
@joperezr we need this for the VMR. Can you please approve and merge? Is dependency flow automatically running?

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -141,21 +141,15 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>d82d6eb7c6aa6c737a83cbf3141d19db02a00006</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.XliffTasks" Version="1.0.0-beta.23526.1">
-      <Uri>https://github.com/dotnet/xliff-tasks</Uri>
-      <Sha>8523ee2147c37df79a26122ad11a9e29fbededf2</Sha>
+    <Dependency Name="Microsoft.DotNet.XliffTasks" Version="9.0.0-beta.24101.1">
+      <Uri>https://github.com/dotnet/arcade</Uri>
+      <Sha>d82d6eb7c6aa6c737a83cbf3141d19db02a00006</Sha>
```
Comment (joperezr) [inline eng/Version.Details.xml:146]: Do we not need the SourceBuild attribute here though?

**Review by joperezr — APPROVED** [review]
Left one comment, but looks good otherwise

File: eng/Version.Details.xml (hunk 1 above)
Comment (akoeplinger) [reply eng/Version.Details.xml:146]: no we don't, this is just like any of the other packages which come from arcade

**Review by joperezr — APPROVED** [review]

**Comment by joperezr** [comment]
Thanks @akoeplinger!

**Pull request merged**